handlers/handler.go

```diff
@@ -95,6 +96,56 @@ func (env *Env) PostUserHandler(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(reqUser)
 }
 
+// GetUserHandler gets a user returning specified columns
+func (env *Env) GetUserHandler(w http.ResponseWriter, r *http.Request) {
+	userID, err := strconv.ParseInt(r.Header.Get("User-ID"), 10, 64)
+	if err != nil {
+		errMsg := "Invalid user ID"
+		log.Println(errMsg + ": " + err.Error())
+		http.Error(w, errMsg, http.StatusBadRequest)
+		return
+	}
+
+	user, err := env.DB.ReadUser(userID)
+	if user == nil {
+		errMsg := "User-ID not found"
+		log.Println(errMsg + ": " + err.Error())
+		http.Error(w, errMsg, http.StatusNotFound)
+		return
+	}
+	if err != nil {
+		internalServerError(w, err)
+		return
+	}
+
+	responseUser := &models.User{}
+	r.ParseForm()
+	includes := r.Form["includes"]
+	if includes != nil {
+		for _, column := range includes {
+			if column == "id" {
+				responseUser.ID = user.ID
+			} else if column == "name" {
+				responseUser.Name = user.Name
+			} else if column == "email" {
+				responseUser.Email = user.Email
+			} else if column == "password" {
+				responseUser.Password = user.Password
+			} else if column == "avatar_url" {
+				responseUser.AvatarURL = user.AvatarURL
+			} else {
+				errMsg := "Invalid includes format"
+				http.Error(w, errMsg, http.StatusBadRequest)
+				return
+			}
+		}
+	}
+
+	w.Header().Add("Content-Type", "application/json")
+	w.WriteHeader(http.StatusCreated)
```

Comment:
You can remove this line. This handler should return `200 OK` and it'll do that by default as long as you don't explicitly set the status code.